src/types/visualization.d.ts

```diff
@@ -16,4 +16,5 @@ export interface ChartInput {
     data: DataRecord;
     axisPairs: AxisPair[];
     aspect: number;
+    germanFormat?: boolean;
```

Comment:
As far as I can see, this new option isn't used anywhere.
Once we are at it, maybe lets get rid of this interface the way we did for the `GeoMap` component?